packages/@react-spectrum/overlays/src/Underlay.tsx

```diff
@@ -11,16 +11,19 @@
  */
 
 import {classNames} from '@react-spectrum/utils';
-import React from 'react';
+import React, {Ref} from 'react';
 import underlayStyles from '@adobe/spectrum-css-temp/components/underlay/vars.css';
 
 interface UnderlayProps {
   isOpen?: boolean,
   isTransparent?: boolean
 }
 
-export function Underlay({isOpen, isTransparent}: UnderlayProps) {
+function Underlay({isOpen, isTransparent}: UnderlayProps, ref: Ref<HTMLDivElement>) {
```

Comment:
do we still need the ref here? i am not quite seeing where it coming from or being used

Reply:
Good catch, forgot to remove that